basics/indexing/json-index.md

```diff
@@ -419,6 +448,8 @@ FROM mytable
 WHERE JSON_MATCH(person, '"$.name"=''adam'' AND "$.addresses[*].number"=112')
 ```
 
+`NOT IN` and `!=` can be used in nested filter expressions starting from Pinot 1.2.0. Note that `IS NULL` cannot be used in nested filter expressions currently.
```

Comment:
> `NOT IN` and `!=` can be used in nested filter expressions starting from Pinot 1.2.0

I'm wondering if this needs to be stated explicitly or can be omitted?

Reply:
We can put them in a hint blob. We can phrase it as it was not supported before 1.2.0